example/tutorial/03_simple_view_lambda/CMakeLists.txt

```diff
@@ -2,11 +2,9 @@
 KOKKOS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
 KOKKOS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
 
-IF (Kokkos_ENABLE_CXX11)
-  # This is a tutorial, not a test, so we don't ask CTest to run it.
-  KOKKOS_ADD_EXECUTABLE(
-    tutorial_03_simple_view_lambda
-    SOURCES simple_view_lambda.cpp
-    COMM serial mpi
-    )
-ENDIF ()
+# This is a tutorial, not a test, so we don't ask CTest to run it.
+KOKKOS_ADD_EXECUTABLE(
+  tutorial_03_simple_view_lambda
+  SOURCES simple_view_lambda.cpp
+  COMM serial mpi
```

Comment:
The `COMM` line is not necessary because the tutorial does not use MPI and somehow this trips the tester.